usb/backend/libusb1.py

```diff
@@ -575,6 +582,16 @@ def libusb_fill_iso_transfer(_libusb_transfer_p, dev_handle, endpoint, buffer, l
     #int libusb_handle_events(libusb_context *ctx);
     lib.libusb_handle_events.argtypes = [c_void_p]
 
+    try:
+        lib.libusb_hotplug_register_callback.argtypes = [c_void_p, c_int, c_int, c_int, c_int, c_int, _libusb_hotplug_callback_fn, py_object, POINTER(_libusb_hotplug_callback_handle)]
+    except AttributeError:
```

Comment:
Not really doing a review, just putting a comment: A mention about why exception handling was needed would be helpful

Reply:
I have no idea. :-D

Reply:
Looking at the code: I am not sure, the code existing code above mine also catched the exception - sometimes. Too bad I didn't document this. But I guess I just did the same way as the existing code.
Might have something to do with `POINTER` c_void_p? not sure.

Reply:
In the other places it is used to deal with older versions libusb 1.0 that lack certain APIs.¹

All hotplug APIs are marked with

> Since version 1.0.16, LIBUSB_API_VERSION >= 0x01000102

so the try-catch blocks are appropriate here.

---

_¹ Though in some of these existing places it might not be necessary after all (e.g. `libusb_get_port_number`)._